### DISCUSSION TITLE
TS0601 massive loging daemon.log

### DISCUSSION THREAD

**Discussion by jujoparo** [Koenkk/zigbee2mqtt#17712] [Devices]
Since I have paired a TS0601 human presence detector my daemon.log is getting so big that I must delete it every 3 days because it blocks my raspberry. 

What I have seen is that every zigbee message is loged in daemon.log what I think is a complete nonsense.

How can I disable the logging in that file. Next is an example of what I see in the daemon.log file:

`
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/cli', payload ' '
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/illuminance_lux', payload '3'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/last_seen', payload '2023-05-16T19:44:49.853Z'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/linkquality', payload '65'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/presence', payload 'false'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/maximum_range', payload '6'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/radar_sensitivity', payload '3'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/minimum_range', payload '0.6'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/self_test', payload 'check_success'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/detection_delay', payload '10'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/fading_time', payload '63'
May 16 21:44:49 Dagobah npm[638]: Zigbee2MQTT:info  2023-05-16 21:44:49: MQTT publish: topic 'zigbee2mqtt/SenPresenDorm1/target_distance', payload '0'
`

**Comment by lux73** [comment]
don't know why z2m ist logging into daemon.log File on your System - on my bare Metal System (Debian 11.7) only hostapd wireless Devices (tasmota) is logged into
